### DISCUSSION TITLE
1.7 Chia has been stolen from my wallet.

### DISCUSSION THREAD

**Discussion by unocual** [Chia-Network/chia-blockchain#9215] [Support]
Good morning, I have been checking my chia wallet this morning and a payment of 1.7 chia has been made without my permission at 23:23 at night.

Is there a way to roll back the Transaction?

To be able to get my chia back.

**Comment by mdrodge** [comment]
No. (roll back is not a thing in crypto)
You should review your security and set up a cold wallet with a new mnemonic quickly before they steal any remaining funds you may have.
Keep your keys safe.
